test/model.jl

```diff
@@ -355,4 +355,27 @@ end
     @test value(model, x) ≈ 1.0 atol=1e-6
 end
 
+@testset "Parameter(A, model)" begin
```

Comment:
Should seed the RNG here (or ideally use a separate `MersenneTwister` with a fixed seed) to avoid random test failures.